### PR TITLE
tilix: update to 1.8.7.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by maxice8** [comment]
Travis is failing

**Comment by Cogitri** [comment]
Hum, I have problems even building gtkd right now because `ld.gold` just crashes during linking.

**Comment by maxice8** [comment]
On Fri, Jan 04, 2019 at 10:04:50AM -0800, Rasmus Thomsen wrote:
> Hum, I have problems even building gtkd right now because `ld.gold` just crashes during linking.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/6585#issuecomment-451521226

force usage of ld.bfd ?

**Comment by Cogitri** [comment]
Yup, took a sec to figure out how ldc does this since it doesn't take LD

**Comment by Cogitri** [comment]
Should work now, let's see if CI likes it...

File: srcpkgs/tilix/patch [hunk 1]
```diff
@@ -0,0 +1,79 @@
+--- /home/rasmus/Downloads/tilix-1.8.7//source/gx/tilix/terminal/terminal.d	2019-01-04 01:03:40.000000000 +0100
```
Comment (maxice8) [inline srcpkgs/tilix/patch]: this seems wrong.

**Pull request closed without merging**